azure/services/networkinterfaces/client.go

```diff
@@ -53,44 +62,114 @@ func newInterfacesClient(subscriptionID string, baseURI string, authorizer autor
 	return nicClient
 }
 
-// Get gets information about the specified network interface.
-func (ac *AzureClient) Get(ctx context.Context, resourceGroupName, nicName string) (network.Interface, error) {
-	ctx, _, done := tele.StartSpanWithLogger(ctx, "networkinterfaces.AzureClient.Get")
+// Get gets the specified network interface.
+func (ac *AzureClient) Get(ctx context.Context, spec azure.ResourceSpecGetter) (result interface{}, err error) {
+	ctx, _, done := tele.StartSpanWithLogger(ctx, "interfaces.AzureClient.Get")
```

Comment:
Shouldn't the logger span still be labeled with `networkinterfaces`? I thought it was always "packagename.struct.func".

Reply:
Yeah it should, thanks for catching that!